src/index.html

```diff
@@ -649,14 +653,14 @@ <h1 id="embed-and-inject">Embed and Inject</h1>
       <form name="myForm">
         <div class="control-group" ng-class="{error: myForm.name.$invalid && !myForm.name.$pristine}">
           <label>Name</label>
-          <input type="text" name="name" ng-model="project.name" required>
+          <input type="text" name="name" ng-model="editProject.project.name" required>
           <span ng-show="myForm.name.$error.required && !myForm.name.$pristine" class="help-inline">
               Required {{myForm.name.$pristine}}</span>
```

Comment:
What purpose is `{{myForm.name.$pristine}}` supposed to serve here ?
Is it a typo ?